### ISSUE TITLE
Better layout for use cases pages.

### ISSUE THREAD

**Issue by Comandeer** [ckeditor/editor-recommendations#8]
I've pushed [first draft of "Blog/Article" use case](http://ckeditor.github.io/editor-recommendations/use-cases/blog-article.html), but honestly I don't like it very much. There must be some better way to do it.

Currently there are 3 sections:
- "Structure" that describes how such type of content is created (with examples)
- "Additional information" that describes known problems or other things worth noting about that type of content
- "Guidelines for editor" that… gives guidelines for editor ;)

It's the most sensible way of creating use case page that I came up. But I don't think it's sensible enough to stay.

**Comment by fredck** [comment]
My original concept for the Editor Recommendations project was to satisfy the "I don't want to think about this, just solve my problem" reality. In fact, many people out there simply pick ready to use solutions, lacking knowledge and/or time to go on details.

Taking the above in consideration, the deliverable os this project should be "simple" and, unlike W3C standards, no one would be ever forced to follow them. You could have variations on implementations based on this project, which should serve simply as a good "starting point".

When it comes to "Use Cases", this should be the most important deliverable of this project: tell people what is the recommended editor to use on specific cases (not tell me what is the use case). In few words, don't make me things and just show me what you recommend me. I'll then do whatever I want with it.

Another outcome of the project is details on "Features" and "Usability (Features)". This should give more information about aspects which are usually common to several different Use Cases.

Therefore we could summarize this in this way: `Use Case = Features + Usability`. So to make it simple, we could simply list which Features and Usability are part of our recommendation to a Use Case.

We don't need much talking and explanations in our recommendations. They can be short and straight. We don't need "shoulds" and "musts". It is implicit that nothing here is a must. In the other hand, we may have a few "notes" that are useful for people to decide whether or not one of the recommended features make sense for their case. For example, if the "table" feature should be used or not. But these should be rare cases. For instance, the "image" feature is pretty obvious so one case easily decide to drop it from our recommendation.

Summing up all the above, I would propose the following structure for Use Cases:
- **Use Case**
  - (Description): may be obvious, but needed for clarity.
  - **Features**: list of Features (linked to their details page) and Usability Features (also linked). Later on we could add a screenshot of an editor with this setup.
  - **Notes**: options and additional aspects to be taken in consideration. In other words, anything that can be useful.

I'm afraid that the current proposal for the Use Case is mostly focused on describing what is the case and not what is the editor recommendation.

**Comment by Comandeer** [comment]
I've updated [draft for "Blog/Article" use case](http://ckeditor.github.io/editor-recommendations/use-cases/blog-article.html).
- The description of use case is limited to the minimum.
- List of features are divided into "Features" and "Usability Features" subcategories.
- There are also "Notes", "Sources of images" and "References".

I deleted 2 of images, but keep 2 other. I think that images inside recommendations could clarify many things. Looking at W3C specs proves my point ;)

> We don't need "shoulds" and "musts".

I'm not so sure about it. As an administrator of forum about webdeveloping, I clearly see that people tend to do some very stupid things (e.g. deleting password hashing just because… they can). Moreover there is no sensible way to tell them that there is some simpler way or that they are just wrong. They just want to know **HOW** to do what they want to do without even trying to know **WHY** it should be done the other way. Maybe I'm a little bit pessimistic, but I have a feeling that most of people wouldn't even bother with reading our recommendations. They'd just look for "Download" button for editor and… that's it. I believe that ER will be read mostly by people, who'd like to know **WHY**, because they already know **HOW**.

And in this case "should" and "must" are obviously not needed. On the other hand: maybe we could really convince someone how it should be done correctly… SHOULD/MUST is always better and more "aggresive" than normal words in that cases.

But apparently I forgot about my own rules while creating the first draft for "Blog/Article" use case, because you're right – there was too much "how" and very little "why".

**Comment by fredck** [comment]
> Maybe I'm a little bit pessimistic, but I have a feeling that most of people wouldn't even bother with reading our recommendations.

Note that the main target for this recommendations is: "us". Other editor developers may also benefit of it.

The whole point is for us to have enough research around features and use cases so CKEditor can be proposed in the best possible way. By doing this here, we are able to have input from the community so we don't take the whole responsibility.

Other than that, this may help us in the future when justifying our decisions. If one would ever ask "whys", we'll have where to point our finger to.

**Comment by Comandeer** [comment]
Probably we can close this issue for now. Use cases pages are readable enough.
